packages/frontend-api/src/Model/Resolver/Article/ArticleQuery.php

```diff
@@ -50,27 +50,27 @@ public function articleByUuidOrUrlSlugQuery(?string $uuid = null, ?string $urlSl
     }
 
     /**
-     * @return array
+     * @return string
      */
-    public function termsAndConditionsArticleQuery(): array
+    public function termsAndConditionsArticleUrlQuery(): string
```

Comment:
change of return type should be mentioned in the backend part of upgrade notes as a backward incompatible change

Reply:
Done, thanks :)